### PR TITLE
Configuration to show/hide original price on course landing page

### PR DESCRIPTION
**Pull request by asadiqbal08**
**Description:** Configuration to show/hide original price on course landing page.

**JIRA:** [ENT-966](https://openedx.atlassian.net/browse/ENT-966)

### CODE REVIEW

**Comment by zubair-arbi** [comment]
@asadiqbal08 this is what we wanted for ENT-966.

File: tests/test_enterprise/views/test_course_enrollment_view.py [hunk 1]
```diff
@@ -205,6 +205,81 @@ def test_get_course_enrollment_page(
         response = self.client.get(enterprise_landing_page_url)
         self._check_expected_enrollment_page(response, expected_context)
 
+    @mock.patch('enterprise.views.render', side_effect=fake_render)
+    @mock.patch('enterprise.api_client.lms.embargo_api')
+    @mock.patch('enterprise.views.CourseCatalogApiServiceClient')
+    @mock.patch('enterprise.views.EnrollmentApiClient')
+    @mock.patch('enterprise.api_client.ecommerce.ecommerce_api_client')
+    @mock.patch('enterprise.utils.Registry')
+    def test_hide_course_original_price_value_on_enrollment_page(
```
Comment (zubair-arbi) [inline tests/test_enterprise/views/test_course_enrollment_view.py]: Please combine both tests by using ddt for default value of `hide_course_original_price` in `EnterpriseCustomerFactory`.

**Comment by UmanShahzad** [comment]
@asadiqbal08 Did you tag 0.68.5? I don't see it on pypi. I've made a PR for 0.68.4 though: https://github.com/edx/edx-platform/pull/18219

**Comment by asadiqbal08** [comment]
@UmanShahzad I am going with a new release for 0.68.5

**Pull request merged**